Automattic/jetpack
# CRM: Emerald polish

## Proposed changes:
* This PR addresses the last touches for our Emerald Project.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3132

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:

* Go through the following list checking each change:

Things to consider:
* on small screen, title wraps, buttons wrap onto title (see transactions listview)
* black `jpcrm-button` has blue text on focus if `<a>`
* blue on `/wp-admin/admin.php?page=zerobscrm-team`
* blue on `/wp-admin/admin.php?page=zerobscrm-datatools`
* blue in `zbs-*-learn-nav`, `zbs-*-quicknav` (add contact or company to object)

Listview considerations:
* missing top pagination (for long results pages)
* sort and pagination use links instead of AJAX calls
* update "no results" and "loading" content
